build/cpp/cmd/src/cmd.cc

```diff
@@ -92,6 +92,12 @@ void cmd::CommandLine::Warble(const std::string &username,
   Any any;
   request.set_username(username);
   request.set_text(text);
+  srand(time(NULL));
+  int rand_id = rand();
+  while (CheckWarble(rand_id)) {
+    rand_id = rand();
+  }
+  request.set_id(std::to_string(rand_id));
```

Comment:
Seeds random number at current time. Creates new random integers until there is a free random integer. This integer becomes the id of the warble request.

Reply:
This might be a matter of preference, but I feel the ID assignment to the warble should happen within the warble code itself.

Reply:
I had it in the warble code; however, I moved it here, so I could make calls to the server to check if a user with the ID already existed.